### DISCUSSION TITLE
[BABEL] Note: The code generator has deoptimised the styling...

### DISCUSSION THREAD

**Discussion by npearson72** [pmndrs/jotai#2050] [General] [answered]
I'm getting the following log on build:

```
[BABEL] Note: The code generator has deoptimised the styling of /Users/me/project/node_modules/jotai-devtools/dist/chunk-6WBHWUYQ.esm.mjs as it exceeds the max of 500KB.
```

Is this something that we should worry about?

Can it be fixed or silenced?

**Comment by arjunvegda** [comment, marked as answer]
> Is this something that we should worry about? Can it be fixed or silenced?

This could be safely ignored. However, you should be able to silence it using Babel's `compact` option.  See: https://github.com/babel/babel/discussions/13676#discussioncomment-1183149

We plan to resolve it by chunking the `jotai-devtools` bundle further. Last I checked, ESBuild blocked us, it did not support chunking.

**Reply by npearson72** [reply]
Thanks you.
